MAINTAINERS.yml

```diff
@@ -481,6 +481,7 @@ Documentation:
   status: maintained
   maintainers:
     - carlescufi
+    - kartben
```

Comment:
You should add yourself as the first maintainer in the list so that the CI assigns any related PRs to you.